### DISCUSSION TITLE
Welcome to Kronos Discussions!

### DISCUSSION THREAD

**Discussion by TrevorSatori** [TrevorSatori/kronos#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lautarodragan** [comment]
Hey! Just wanted to say I'm loving Kronos so far. I started using it yesterday. Downloaded the compiled binary from the v0.69 release.

Just now I `git clone`'d the project, installed rust and cargo and whatnot, and am now running it from `cargo run --release` just fine. 

Thanks for building and sharing this!

I have zero experience with Rust, but hopefully I'll be able to contribute a bit.

I've been using `cmus` for a year or two, and I certainly miss its features, but I can't stand how it needs `apt install`'ing a bunch of dependencies to be able to run it, and haven't been able to support `flac` after I moved from Ubuntu 22 to 24. 

On that note, I do have one question: does the Kronos build process yield a statically-linked, fully functioning binary that can run in any Linux distro without any external dependencies? It does run for me now, but I did install a bunch of dev apt dependencies just yesterday (all the ones required by `cmus`, actually) and am wondering if they impact this thing at all.

**Comment by TrevorSatori** [comment]
Hey, thanks for reaching out, I’m glad you like the project. In the test
builds I use, the only thing I have to install is rust and libasounds2-dev
which comes with most ready made distros to my knowledge.
